### PR TITLE
Add helper script for regenerating test JSON output

### PR DESCRIPTION
**Pull request by rubensworks**
This PR adds the helper script for regenerating JSON files.
I've also regenerated all files to minimize future diffs.

### CODE REVIEW

File: test/generate-json.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import * as fs from 'fs';
+import translate from '../lib/sparqlAlgebra';
+import Util from '../lib/util';
+import * as Path from 'path';
```
Comment (joachimvh) [inline test/generate-json.ts:4]:
```suggestion
import * as path from 'path';
```
I know you like using `Path`, but several other files already use `path` so let's use that one 😄

File: test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json [hunk 2]
```diff
@@ -4,9 +4,12 @@
     "type": "bgp",
     "patterns": [
       {
+        "type": "pattern",
+        "termType": "Quad",
+        "value": "",
```
Comment (joachimvh) [inline test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json:9]: So apparently the `Util.objectify` method adds these values. Can we update that function to not have a value there? I don't think there's a reason to have it? Unless that randomly breaks something.

File: test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json [hunk 3]
```diff
@@ -4,9 +4,12 @@
...
         "subject": {
           "termType": "Variable",
-          "value": "b0"
+          "value": "g_0"
```
Comment (joachimvh) [inline test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json:12]: Why did all these change, is this a SPARQL.js thing?

**Review by joachimvh — CHANGES_REQUESTED** [review]

File: test/generate-json.ts (hunk 1 above)
Comment (rubensworks) [reply test/generate-json.ts:4]: Ah, now I remember why I always use `Path`: it clashes with the local variable name `path`, which tends to be used quite often.
But ok, I'll just rename that variable here in this case :-)

File: test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json (hunk 3 above)
Comment (rubensworks) [reply test/algebra-blank-to-var/dawg-syntax/bnodes/syntax-bnodes-02.json:12]: Yeah, I think there was a change in datafactory in SPARQL.js. In any case, the labels shouldn't matter, as long as they are unique within the query (which they still are).

**Review by joachimvh — APPROVED** [review]

**Pull request merged**